Cargo.toml

```diff
@@ -1,7 +1,9 @@
 [package]
 name = "zip-password-finder"
 version = "0.5.0"
+description = "Find the password of protected ZIP files"
```

Comment:
The description field is already present line 7
However I prefer yours so you can delete the older one :)

Reply:
Done!